libcxx/test/benchmarks/numeric/rand.uni.int.bench.cpp

```diff
@@ -0,0 +1,36 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// UNSUPPORTED: c++03
+
+#include <benchmark/benchmark.h>
+#include <cstdint>
+#include <random>
+
+template <typename Eng, std::uint64_t Max>
+static void bm_rand_uni_int(benchmark::State& state) {
```

Comment:
Let's name this `bm_uniform_int_distribution` so the benchmark names are clearer when running the benchmark. WDYT?

Reply:
That works for me. I had just used this name to match the file's name, but I agree that using the full class name makes more sense here.